Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -585,7 +585,29 @@ int CLuaEngineDefs::EngineRequestModel(lua_State* luaVM)
                     CClientModel* pModel = m_pManager->GetModelManager()->FindModelByID(iModelID);
                     if (pModel == nullptr)
                         pModel = new CClientModel(m_pManager, iModelID, eModelType);
-                    pModel->Allocate();
+
+                    ushort usParentID = -1;
+
+                    if (argStream.NextIsNumber())
+                        argStream.ReadNumber(usParentID);
+                    else
+                    {
+                        switch (eModelType)
+                        {
+                            case eClientModelType::PED:
+                                usParentID = 7;
+                                break;
+                            case eClientModelType::OBJECT:
+                                usParentID = 1337;
+                                break;
+                            case eClientModelType::VEHICLE:
+                                usParentID = 400;
+                                break;
```

Comment:
```suggestion
                        switch (eModelType)
                        {
                            case eClientModelType::PED:
                                usParentID = 7; // male01
                                break;
                            case eClientModelType::OBJECT:
                                usParentID = 1337; // BinNt07_LA (trash can)
                                break;
                            case eClientModelType::VEHICLE:
                                usParentID = VT_LANDSTAL;
                                break;
```
Please format this code using the formatter to fix the code comment indentation.